<launch>
 <node pkg="tf" type="static_transform_publisher" name="fftip_transform_pub"
    args="0.0 0.0 0.025 0.0 0.0 0.0 /ffdistal /fftip 50" />
 <node pkg="tf" type="static_transform_publisher" name="mftip_transform_pub"
    args="0.0 0.0 0.025 0.0 0.0 0.0 /mfdistal /mftip 50" />
 <node pkg="tf" type="static_transform_publisher" name="rftip_transform_pub"
    args="0.0 0.0 0.025 0.0 0.0 0.0 /rfdistal /rftip 50" />
 <node pkg="tf" type="static_transform_publisher" name="lftip_transform_pub"
    args="0.0 0.0 0.025 0.0 0.0 0.0 /lfdistal /lftip 50" />
 <node pkg="tf" type="static_transform_publisher" name="thtip_transform_pub"
    args="0.0 0.0 0.0275 0.0 0.0 0.0 /thdistal /thtip 50" />

 <node pkg="sr_utilities" type="fingertip_pos_publisher.py" name="fftip_pos_pub" />

</launch>
